workspaces/orchestrator/README.md

```diff
@@ -1,13 +1,10 @@
 # [Backstage](https://backstage.io)
 
-This is your newly scaffolded Backstage App, Good Luck!
+## Orchestrator documentation
 
-To start the app, run:
```

Comment:
I think this should stay
In general I think this README should be for the developer of the orchestrator plugins in rhdh-plugins repo and the readme in ./plugins/orchestraotr/README.md should be for cunsomers of the orchestrator plugin as static or dynamic plugin
Currently there are details in ./plugins/orchestrator/README.md related to openapi development that shouldn't be there, but we shouldn't add more